addons/auth_signup/tests/test_login.py

```diff
@@ -1,2 +1,21 @@
-# rerun TestWebLogin tests with auth_signup installed
-from odoo.addons.web.tests.test_login import TestWebLogin  # pylint: disable=W0611
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.web.tests.test_login import TestWebLogin
+from odoo.tests.common import new_test_user, tagged
+
+
+@tagged('post_install')
+class TestWebPublicLogin(TestWebLogin):
+    """Rerun TestWebLogin tests with auth_signup installed + additional ones. """
+    def test_web_login_public_user(self):
+        self.public_user = new_test_user(
+            self.env,
+            name='Public user',
+            login='public_user',
+            groups='base.group_public',
```

Comment:
shouldn't you make a portal user instead ? (it's not realist to login with a public user)